packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SubTree/PublicCloudPanel.tsx

```diff
@@ -120,7 +119,7 @@ export const PublicCloudPanel: React.FC<ComponentProps<
   useEffect(() => {
     if (
       selectedPciProject &&
-      rootNode.id === publicCloud.id &&
+      rootNode.id === 'pci' &&
```

Comment:
As we talked about, maybe put this in a constant file and reuse it where needed (here and packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/publicCloud.ts)